File: exerciciosC/calculadora.c
#include <stdio.h>
float so(float a, float b){
    float s;
    s = (a+b);
    return(s);
}
float su(float a, float b){
    float s;
    s = (a-b);
    return (s);
}
float m(float a, float b){
    float r;
    r = (a*b);
    return(r);
}
float d(float a, float b){
    float r;
    r = (a/b);
    return(r);
}
int main(){
    float a,b,r;
    int escolha;
    do{
    printf("\n---------------------\n");
    printf("1 - Subtracao \n");
    printf("2 - Soma \n");
    printf("3 - Multiplicacao \n");
    printf("4 - Divisao \n");
    printf("5 - Sair do programa \n");
    printf("---------------------\n");
    printf("\nEscolha uma opcao: ");
    scanf("%d",&escolha);
    if(escolha==5){
        printf("\n-----------------------------------\n");
        printf("Voce escolheu sair do programa. . .");
        printf("\n-----------------------------------\n");
        break;
    }
    else{
        printf("Digite um numero: ");
        scanf("%f",&a);
        printf("Digite outro numero: ");
        scanf("%f",&b);
    }
    switch(escolha){
    case 1:
        r = su(a,b);
        printf("\nO resultado eh: %.2f",r);
        break;
    case 2:
        r = so(a,b);
        printf("\nO resultado eh: %.2f",r);
        break;
    case 3:
        r = m(a,b);
        printf("\nO resultado eh: %.2f",r);
        break;
    case 4:
        if(b==0){
            printf("\n----------------------------\n");
            printf("ERRO! nao pode dividir por 0");
            printf("\nTente novamente!");
            printf("\n----------------------------\n");
            break;
        }
        else{
        r = d(a,b);
        printf("\nO resultado eh: %.2f",r);
        }
        break;
    default:
        printf("\nOpcao invalida !");
        break;
    }
    }while(escolha!=5);
}
//~~phelipes2000

File: Exercicios/Python/calculadora.py
def so(a,b):
    return a+b

def su(a,b):
    return a-b

def m(a,b):
    return a*b

def d(a,b):
    return a/b

def main():
    print("----------------------")
    print("1 | [Soma]")
    print("2 | [Subtracao]")
    print("3 | [Divisao]")
    print("4 | [Multiplicacao]")
    print("5 | [Sair do programa]")
    print("----------------------")
    print
    i = int(input("Escolha uma opção:"))
    if i <=0 or i >=6:
        print("")
        print("Opcao invalida. Tente novamente!")
        print("")
    elif i == 5:
        print("")
        print("Voce escolheu sair do programa . . .")
        print("")
    else:
        print("")
        x = int(input("Digite um numero: "))
        y = int(input("Digite outro numero: "))
        if i == 1:
            r = so(x,y)
        if i == 2:
            r = su(x,y)
        if i == 3:
            r = d(x,y)
        if i == 4:
            r = m(x,y)
        print(" ")
        print("-----------------")
        print(f'O resultado eh: {r}')
        o = input("Tecle [ENTER]")
        main()

main()
//~~phelipes2000